pandas/tests/test_frame.py

```diff
@@ -12053,6 +12054,60 @@ def test_quantile(self):
         expected = Series([3., 4.], index=[0, 1])
         assert_series_equal(result, expected)
 
+    def test_quantile_interpolation(self):
```

Comment:
can you add here a link to the github issue in a comment (first line of the function) (just `# GH...`)